opcua/common/xmlparser.py

```diff
@@ -331,6 +331,8 @@ def _parse_body(self, el):
         return body
 
     def _parse_refs(self, el, obj):
+        parent, parentlink = obj.parent, None
```

Comment:
I trust you that you analysis is correct but that local variable assignment seems a bit broken or simply overcomplicated. you create a parent variable but then still use obj.parent at several places.. I cannot follow it and I suppose it should be possible to simplify this...and generally I am not very fan of creating local variables just to reduce variable length..

Reply:
`parent`  and `parentlink`  are the fallbacks that will be used if no other backward reference matches (-> after the loop).

Reply:
> but then still use obj.parent at several places.. 

only to set it.

Reply:
`parent`  gets initialized to `obj.parent` for the case that there are no backward references at all.
This way it will stay the same, if `obj.parentlink`  is `None`.